src/condor_ce_run

```diff
@@ -164,7 +164,7 @@ def check_remote_status(job_info):
     status_re = re.compile(r"(\d+)")
     for line in stdout:
         try:
-            job_status = int(status_re.match(line).group(1))
+            job_status = int(status_re.match(str(line)).group(1))
```

Comment:
```suggestion
            job_status = int(status_re.match(line).group(1))
```
Either `line` is already a str (in which case str() is unnecessary) or it's a bytes (in which case str() is wrong because it would surround the result with `b''`).